window/src/os/windows/wgl.rs

```diff
@@ -173,7 +191,7 @@ impl GlState {
             }
 
             let extensions = if ext.GetExtensionsStringARB.is_loaded() {
-                unsafe { cstr(ext.GetExtensionsStringARB(hdc as *const _)) }
+                unsafe { cstr(ext.GetExtensionsStringARB(std::mem::transmute(hdc))) }
```

Comment:
in general, it is best to avoid `transmute` and take great care with it, because it will let you cast pretty much anything to anything without complaint, and is a great way to mask serious problems.  So please find a way to make this work without an inline `transmute` in here!

Reply:
(and same deal for other uses of `transmute` throughout!)